collaborations/blood-hunter-2022/bloodhunter-mutant.xml

```diff
@@ -0,0 +1,327 @@
+			<h5 class="caption">Mutagencraft</h5>
+			<table>
+				<thead>
+					<tr><td class="col-5">Blood Hunter Level</td><td class="col-15">Mutagens Created</td><td class="col-20">Formulas Known</td></tr>
+				</thead>
+				<tr><td> 3rd</td><td>1</td><td>4</td></tr>
+				<tr><td> 7th</td><td>2</td><td>5</td></tr>
+				<tr><td> 11th</td><td>2</td><td>6</td></tr>
+				<tr><td> 15th</td><td>3</td><td>7</td></tr>
+				<tr><td> 18th</td><td>3</td><td>8</td></tr>
+			</table>
+		</description>
+		<sheet action="Bonus Action">
+			<description>You can consume a single mutagen, and the effects and side effects last until you finish a short or long rest, unless otherwise specified. While one or more mutagens are affecting you, you can use an action to focus and flush the toxins from your system, ending the effects and side effects of all mutagens.		
+			Mutagens have no effect on other creatures. They also lose their potency and becoming inert if not used before you finish your next short or long rest.
+			You can concoct a single mutagen when you finish a short or long rest. </description>			
+			<description level="7">You can consume a single mutagen, and the effects and side effects last until you finish a short or long rest, unless otherwise specified. While one or more mutagens are affecting you, you can use an action to focus and flush the toxins from your system, ending the effects and side effects of all mutagens.		
+			Mutagens have no effect on other creatures. They also lose their potency and becoming inert if not used before you finish your next short or long rest.
+			You can concoct two mutagens when you finish a short or long rest.</description>
+			<description level="15">You can consume a single mutagen, and the effects and side effects last until you finish a short or long rest, unless otherwise specified. While one or more mutagens are affecting you, you can use an action to focus and flush the toxins from your system, ending the effects and side effects of all mutagens.		
+			Mutagens have no effect on other creatures. They also lose their potency and becoming inert if not used before you finish your next short or long rest.
+			You can concoct three mutagens when you finish a short or long rest.</description>
+		</sheet>
+	</element>
+	
+	<element name="Formulas" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_MUTANT_FORMULAS">
+		<description>
+			<p>The number of mutagens you can concoct when you finish a rest, and the number of formulas you know, increases as you gain levels in the blood hunter class, as shown on the Mutagencraft table above. Additionally, when you learn a new mutagen formula, you can replace one formula you already know with a new mutagen formula. You choose four mutagen formulas to learn from the options detailed at the end of this subclass description, and you can concoct one mutagen when you finish a short or long rest.</p>
+		</description>
+		<sheet>
```

Comment:
`display="false"`

Reply:
Warlock invocations and artificer infusions keep the name of the feature on the sheet.